### ISSUE TITLE
wells fargo system parameters move to app.ini

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1631]
_Imported from trac ticket 1631.
Created by **kbower**
Opened in trac: May 7, 2012, 10:06:56 AM
Last modified in trac: May 7, 2012, 10:31:23 AM_
***
_by kbower:_
 * The system parameters are still referenced from the orders.py controller.
```python
  creditapp.merchantNumber = SystemParameter.value(u'wells_fargo_merchantid')
  creditapp.userName = SystemParameter.value(u'wells_fargo_username')
  creditapp.setupPassword = SystemParameter.value(u'wells_fargo_password')
``` * The system parameters are still reloaded each time from `eld.py` customerdata.  (They need to be deleted).
 * The eld.py dataupdate needs to be updated to run again:
```python
    2600: [
        [False, "tables['systemparameters'].delete(tables['systemparameters'].c.name==u'wells_fargo_merchantid')"],
        [False, "tables['systemparameters'].delete(tables['systemparameters'].c.name==u'wells_fargo_username')"],
        [False, "tables['systemparameters'].delete(tables['systemparameters'].c.name==u'wells_fargo_password')"],
        ],
```

**Comment by kbower** [comment]
_**kbower** trac comment on May 7, 2012, 10:16:33 AM:_

(Changeset [[2626]](../commit/5d3a323eced2661bbc89399e4181ba2ef8ce794f))
 * fixes #1631 (move wells fargo system parameters to app.ini so train doesn't risk adopting live values)

**Comment by kbower** [comment]
_**kbower** trac status change on May 7, 2012, 10:16:33 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on May 7, 2012, 10:25:24 AM:_

Still need stub values in `app.ini.template`

**Comment by kbower** [comment]
_**kbower** trac status change on May 7, 2012, 10:25:24 AM:_ **reopened**

**Comment by kbower** [comment]
_**sshah** trac comment on May 7, 2012, 10:31:23 AM:_

(Changeset [[2628]](../commit/b5eb4a80d88077f7d142deedfb9567d5ffa483e2))
 * Fixes #1631 added stub values in app.ini.template

**Comment by kbower** [comment]
_**sshah** trac status change on May 7, 2012, 10:31:23 AM:_ **closed**
